src/Helper/TcaHelperTrait.php

```diff
@@ -113,6 +113,65 @@ protected function hasKeyValuePair(Array_ $configValueArray, string $configKey,
         return false;
     }
 
+    /**
+     * Check if an item exists in a comma-separated list of items.
+     *
+     * @param string $list Comma-separated list of items (string)
+     * @param string $item Item to check for
+     * @return bool TRUE if $item is in $list
+     */
+    protected static function inList(string $list, string $item): bool
+    {
+        return str_contains(',' . $list . ',', ',' . $item . ',');
+    }
+
+    /**
+     * Explodes a string and removes whitespace-only values.
+     *
+     * If $removeEmptyValues is set, then all values that contain only whitespace are removed.
+     *
+     * Each item will have leading and trailing whitespace removed. However, if the tail items are returned as a single
+     * array item, their internal whitespace will not be modified.
+     *
+     * @param non-empty-string $delim Delimiter string to explode with
+     * @param string $string The string to explode
+     * @param bool $removeEmptyValues If set, all empty values will be removed in output
+     * @param int $limit If limit is set and positive, the returned array will contain a maximum of limit elements with
+     * the last element containing the rest of string. If the limit parameter is negative, all components
+     * except the last -limit are returned.
+     * @return string[] Exploded values
+     */
+    protected static function trimExplode(
```

Comment:
This method already exists in the ArrayUtility class. Could you use it from there.